pkg/products/mcg/reconciler.go

```diff
@@ -24,6 +24,7 @@ import (
 	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	k8serr "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/api/resource"
+	k8sresource "k8s.io/apimachinery/pkg/api/resource"
```

Comment:
Minor but if we are opting for naming `k8sresource` could we rename original uses of `resource` instead of importing twice.

Reply:
Thanks Adam, going to leave as `resource`.